components/brave_ads/core/internal/ads_impl.h

```diff
@@ -202,6 +204,8 @@ class AdsImpl final : public Ads,
 
   UserReactions user_reactions_;
 
+  Wallet initial_wallet_;
```

Comment:
Can we please add a comment to state this is a temporary fix until 1.54.x lands which has a more robust solution